data/binding/webstring.go

```diff
@@ -59,3 +44,24 @@ func (s *webSocketString) Close() error {
 
 	return s.conn.Close()
 }
+
+func (s *webSocketString) Get() (string, error) {
+	if s.prev != nil {
+		err := s.prev
+		return "", err
```

Comment:
```suggestion
		return "", s.prev
```

Reply:
or if you don't want to access the field twice;

```go
if e := s.prev; e != nil {
    return "", e
}
```

Reply:
Nice, thanks